midea_beautiful/appliance.py

```diff
@@ -220,12 +222,12 @@ class DehumidifierAppliance(Appliance):
         b"\x10\x02": "fan_speed",
         b"\x14\x02": "mode",
         b"\x17\x02": "filter",
-        b"\x1D\x02": "pump",
-        b"\x1E\x02": "ion",
-        b"\x1F\x02": "auto",
+        b"\x1d\x02": "pump",
```

Comment:
These byte changes look weird but sounds like from a-f are case insensitive from looking and tested with

```python
❯ python 
Python 3.13.0 (main, Oct 16 2024, 03:23:02) [Clang 18.1.8 ] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> byte_string_upper = b"\x1D\x02"
... byte_string_lower = b"\x1d\x02"
... 
... print(byte_string_upper)
... print(byte_string_lower)
... print(byte_string_upper == byte_string_lower)
... 
b'\x1d\x02'
b'\x1d\x02'
True
```